packages/contracts-bedrock/snapshots/state-diff/Deploy.json

```diff
@@ -0,0 +1,25 @@
ses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\",\"{\"storageAccesses\":\"storageAccesses\"}\"]"
```

Comment:
This looks wrong; According to `LibStateDiff`, the serialization should be of the schema:
```json
{
    "accountAccesses": [
        {
            "account": "...",
            "accessor": "...",
            "oldBalance": ...,
            "newBalance": ...,
            "deployedCode": "...",
            "value": ...,
            "data": "...",
            "reverted": ...,
            "chainInfo": { ... },
            "accountAccessKind": { ... },
            "storageAccesses": { ... },
        },
        ...
    ]
}
```

Reply:
Do you have an idea for how to get it to work?